data-reporting-service/investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/config/KafkaStreamsConfig.java

```diff
@@ -18,8 +19,7 @@ public class KafkaStreamsConfig {
     @Value("${spring.kafka.bootstrap-servers}")
     private String bootstrapServers;
 
-    @Value("${spring.kafka.streams.application-id}")
-    private String applicationId;
+    private String applicationId = UUID.randomUUID().toString();
```

Comment:
This should be a constant id like `investigaiton-reporting-service`. Otherwise autoscaling will create separate clients for each of the randomUUID and process the same data across multiple time over each of the auto scaled service.

Reply:
let me quickly check this as I remember even if it is unique id it will not be an issue during auto scaling.

Reply:
fixed!